IStorageEngines/diceKV.go

```diff
@@ -0,0 +1,16 @@
+package istorageengines
+
+import (
+	"sync"
+
+	"github.com/dicedb/dice/object"
+)
+
+type IKVStorage interface {
+	Put(key string, obj *object.Obj)
+	Get(key string) *object.Obj
+	Del(key string) bool
+	GetCount() uint64
+	GetStorage() *sync.Map
```

Comment:
Let's rename it to Count and Storage ... more go-like. and as discussed, the Storage should return an iterator - a function that upon invocation returns the next KV pair.